nextcord/channel.py

```diff
@@ -689,6 +689,10 @@ async def create_thread(
 
         .. versionadded:: 2.0
 
+        .. note::
+
+            This will raise :exc:`HTTPException` if the guild isn't at least level two boosting yet and you are trying to create a private thread.
```

Comment:
This is technically not true as it checks for the PRIVATE_THREADS feature (which can also be given manually by discord staff). Also yet should be removed here

Reply:
wdym?